### PR TITLE
NXDRIVE-2225: Remove the server URL below the account name in the systray

### PR DESCRIPTION
**Pull request by rom1win**
The server url in the systray has been removed and the Nuxeo icon action
tooltip has been updated.

Also changelog has been updated.

### CODE REVIEW

File: nxdrive/data/qml/Systray.qml [hunk 1]
```diff
@@ -140,42 +140,23 @@ Rectangle {
                     }
 
                     // The current account server URL
-                    ScaledText {
+                    Link {
                         id: accountUrl
                         Layout.maximumWidth: parent.width
                         text: accountSelect.getRole("server_url")
                         font.pointSize: point_size * 0.8
-                        color: mediumGray
```
Comment (BoboTiG) [inline nxdrive/data/qml/Systray.qml:148]: We should keep the color, I think.

File: nxdrive/data/qml/Systray.qml [hunk 2]
```diff
@@ -140,42 +140,23 @@ Rectangle {
...
 
-                // Icon 2: open remote server's URL
-                IconLabel {
-                    icon: MdiFont.Icon.nuxeo
-                    onClicked: api.open_remote_server(accountSelect.getRole("uid"))
-                    tooltip: qsTr("OPEN_SERVER") + tl.tr
-                }
-
-                // Icon 3: open local sync root folder
+                // Icon 1: open local sync root folder
```
Comment (BoboTiG) [inline nxdrive/data/qml/Systray.qml:152]: Increment of 1, not 2. It should be :
```suggestion
                // Icon 2: open local sync root folder
```

File: nxdrive/data/qml/Systray.qml [hunk 3]
```diff
@@ -140,42 +140,23 @@ Rectangle {
...
                 IconLabel {
                     icon: MdiFont.Icon.folder
                     onClicked: api.open_local(accountSelect.getRole("uid"), "/")
                     tooltip: qsTr("OPEN_ROOT_FOLDER").arg(APP_NAME) + tl.tr
                 }
 
-                // Icon 4: open the Direct Transfer window
+                // Icon 2: open the Direct Transfer window
```
Comment (BoboTiG) [inline nxdrive/data/qml/Systray.qml:159]: Same here.

File: nxdrive/data/qml/Systray.qml [hunk 4]
```diff
@@ -184,7 +165,7 @@ Rectangle {
                     opacity: feat_direct_transfer ? 1.0 : 0.5
                 }
 
-                // Icon 5: sub-menu
+                // Icon 3: sub-menu
```
Comment (BoboTiG) [inline nxdrive/data/qml/Systray.qml:168]: Same here.

File: nxdrive/engine/engine.py [hunk 5]
```diff
@@ -556,7 +556,7 @@ def get_remote_url(self) -> str:
                 f"{self.server_url}nxhome/{Options.remote_repo}/@view_home?"
                 "tabIds=USER_CENTER%3AuserCenterNuxeoDrive"
```
[the remark this answers is not in the document]
Comment (rom1win) [reply nxdrive/engine/engine.py:557]: Thanks, I wasn't sure about the JSF part.

File: nxdrive/data/qml/Systray.qml (hunk 1 above)
Comment (BoboTiG) [reply nxdrive/data/qml/Systray.qml:148]: Finally, it has been decided to remove the color.

**Pull request merged**